extensions/wikia/JsonFormat/JsonFormatBuilder.php

```diff
@@ -40,7 +40,8 @@ public function popInfoboxKeyValue( JsonFormatInfoboxKeyValueNode $keyValueNode
 			array_pop($this->jsonStack);
 			$this->currentContainer = $this->jsonStack[sizeof($this->jsonStack)-1];
 		} else {
-			throw new JsonFormatException('Invalid infobox');
+			# log invalid instead of failing
+			\Wikia\Logger\WikiaLogger::instance()->debug('Invalid infobox');
```

Comment:
One thing to consider would be using warning for example - debug seems maybe too lightweight for this issue